tabasco/src/server.rs

```diff
@@ -148,14 +179,34 @@ async fn psa_router(psa_request: web::Path<String>, input_data: String) -> Tabas
     Err(TabascoError::UnimplementedRequestError)
 }
 
-pub fn server(url: String) -> Result<Server, String> {
+#[allow(unused)]
+async fn nitro_router(nitro_request: web::Path<String>, input_data: String) -> TabascoResponder {
+    #[cfg(feature = "nitro")]
+    {
+        let inner = nitro_request.into_inner();
+        if inner.as_str() == "AttestationToken" {
+            nitro::attestation_token(input_data)
+        } else {
+            println!("Tabasco::nitro_router returning unsupported with into_inner:{:?}", inner.as_str());
+            Err(TabascoError::UnsupportedRequestError)
+        }
+    }
+    #[cfg(not(feature = "nitro"))]
+    Err(TabascoError::UnimplementedRequestError)
+}
+
+pub fn server(url: String, debug: bool) -> Result<Server, String> {
+    if debug {
+        DEBUG_MODE.store(true, Ordering::Relaxed);
```

Comment:
Is `Ordering::Relaxed` the right memory ordering to use? ` Ordering::SeqCst` is the most restrictive, forcing a fence to be inserted, whereas `Ordering::Relaxed` is the least restrictive.  I'm not a memory ordering expert though, maybe @ShaleXIONG can comment?

Reply:
Neither am I.

Specifically here, I don't think it matters, because modifications to the values only happen on startup. I will change it to SeqCst, though.